server/src/main/java/org/elasticsearch/index/stats/IndexingPressureStats.java

```diff
@@ -70,6 +71,12 @@ public IndexingPressureStats(StreamInput in) throws IOException {
         this.currentCoordinatingOps = 0;
         this.currentPrimaryOps = 0;
         this.currentReplicaOps = 0;
+
+        if (in.getTransportVersion().onOrAfter(TransportVersions.INDEXING_PRESSURE_DOCUMENT_REJECTIONS_COUNT)) {
+            primaryDocumentRejections = in.readVLong();
+        } else {
+            primaryDocumentRejections = 0;
```

Comment:
I think that traditionally `-1` makes sense for a version where we do not support the functionality.

Reply:
Thx, let me change it